notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -840,6 +840,49 @@ facing a home invader [A sadistic janitor broke into a family's home, killed the
 terrorism [The viewer learns that Elijah was ultimately confined to an institution for the criminally insane for having perpetrated three acts of terrorism.]
 
 
+movie: The One (2001)
+=====================
+
+:: Title
+The One
+
+:: Description
+The One is a 2001 American science fiction action film directed by James Wong,
+written by Wong and Glen Morgan, and starring Jet Li, Delroy Lindo, Carla
+Gugino, and Jason Statham.
+
+Synopsis: A rogue agent travels to parallel realities in order for him to kill
+other versions of himself to become a mythical super-being known as "The One".
+
+:: Date
+2001-11-02
+
+:: References
+https://en.wikipedia.org/wiki/The_One_(2001_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+multiverse [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
+parallel universe [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
```

Comment:
How does this relate to "multiverse"? Parent?

Reply:
Deleted the now deprecated *multiverse* theme.